From: Vince Weaver <vince@deater.net>
Date: Fri Dec 13 15:52:25 2013 -0500
Subject: perf: Document the new transaction sample type
Git-commit: 189b84fb54490ae24111124346a8e63f8e019385
Patch-mainline: v3.13-rc5
References: FATE# 315534
Signed-off-by: Tony Jones <tonyj@suse.de

    perf: Document the new transaction sample type
    
    Commit fdfbbd07e91f8fe3871 ("perf: Add generic transaction flags")
    added support for PERF_SAMPLE_TRANSACTION but forgot to add documentation
    for the sample type to include/uapi/linux/perf_event.h
    
    Signed-off-by: Vince Weaver <vincent.weaver@maine.edu>
    Signed-off-by: Peter Zijlstra <peterz@infradead.org>
    Cc: Andi Kleen <ak@linux.intel.com>
    Link: http://lkml.kernel.org/r/alpine.DEB.2.02.1312131548450.10372@pianoman.cluster.toy
    Signed-off-by: Ingo Molnar <mingo@kernel.org>

---
 include/uapi/linux/perf_event.h |    1 +
 1 file changed, 1 insertion(+)

--- a/include/uapi/linux/perf_event.h
+++ b/include/uapi/linux/perf_event.h
@@ -679,6 +679,7 @@ enum perf_event_type {
 	 *
 	 *	{ u64			weight;   } && PERF_SAMPLE_WEIGHT
 	 *	{ u64			data_src; } && PERF_SAMPLE_DATA_SRC
+	 *	{ u64			transaction; } && PERF_SAMPLE_TRANSACTION
 	 * };
 	 */
 	PERF_RECORD_SAMPLE			= 9,
